bitnami/flux/Chart.yaml

```diff
@@ -43,4 +43,4 @@ maintainers:
 name: flux
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/flux
-version: 0.4.0
+version: 0.4.1
```

Comment:
I guess this change was to trigger the tests, right? Are going to revert the bump?

Reply:
Sorry about that, I forgot to remove it after VIB succeeded. Done!